src/dune/super_context.ml

```diff
@@ -651,10 +651,6 @@ module Action = struct
       ~targets_dir ~targets:targets_written_by_user ~expander ~foreign_flags
 end
 
-let opaque t =
```

Comment:
Where did this code go? Was it just useless?

Reply:
(If it was indeed useless, it's worth saying this in the commit message.)

Reply:
It's essentially inside `eval_opaque` now. I'll clarify by documenting opaque.

Reply:
Ah, indeed, not sure how I missed it. 

(I don't see any docs for `opaque` -- are they coming in a separate PR?)

Reply:
Oops, I merged this PR but forgot to push my updates to it. Yes I will just push the doc master now

Reply:
Looks great, thanks!